playwright/tests/screenshots/editor/editor.spec.ts

```diff
@@ -1,6 +1,7 @@
 import type { Page } from '@playwright/test';
 import { test, expect } from '@playwright/test';
 import { ProcessEditor } from '../../page-objects/editor/process-editor';
+import { Inscription } from '../../page-objects/inscription/inscription-view';
```

Comment:
no longer needed right?

Reply:
argh; I'm still damn unfamiliar with VS-Code ... gonna need to add auto-update imports on save, just like in eclipse :dancing_women: